src/main/java/com/spotinst/sdkjava/example/elastigroup/aws/StatefulElastigroupUsageExample.java

```diff
@@ -0,0 +1,109 @@
+package com.spotinst.sdkjava.example.elastigroup.aws;
+
+import com.spotinst.sdkjava.SpotinstClient;
+import com.spotinst.sdkjava.enums.*;
+import com.spotinst.sdkjava.model.RecurrenceFrequencyEnum;
+import com.spotinst.sdkjava.model.*;
+import com.spotinst.sdkjava.model.bl.elastigroup.aws.*;
+import com.spotinst.sdkjava.model.requests.elastigroup.ElastigroupInstanceLockRequest;
+import com.spotinst.sdkjava.model.requests.elastigroup.ElastigroupInstanceUnLockRequest;
+import com.spotinst.sdkjava.model.requests.elastigroup.aws.*;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class StatefulElastigroupUsageExample {
+    private final static String auth_token    = "your-token";
+    private final static String act_id        = "your-account-id";
+
+    private static final String ELASTIGROUP_ID = "stateful-elastigroup-id";
+
+    public static void main(String[] args) throws IOException {
+        // Get elastigroup service client
+        SpotinstElastigroupClient elastigroupClient = SpotinstClient.getElastigroupClient(auth_token, act_id);
+
+        //List stateful instance
+        System.out.println("----------List stateful Instances--------------");
+        List<ElastigroupListStatefulInstancesResponse> statefulInstacesList = listStatefulInstances(elastigroupClient, ELASTIGROUP_ID);
+        String statefulId = null;
```

Comment:
This for loop is redundant, the information is already printed in the called function. No need to retrieve the ID here. We are not doing any operation with it.